base/applications/mspaint/fullscreen.cpp

```diff
@@ -6,12 +6,21 @@
  * PROGRAMMERS: Benedikt Freisen
  */
 
-/* INCLUDES *********************************************************/
-
 #include "precomp.h"
 
+CFullscreenWindow fullscreenWindow;
+
 /* FUNCTIONS ********************************************************/
 
+HWND CFullscreenWindow::DoCreate()
```

Comment:
Why not calling all those "DoCreate" functions, by "Create" instead?

Reply:
The super class has already `Create(...)` method.

Reply:
But which `Create(...)` ? Create without parameters?

Reply:
`CWindowImplBaseT<CWindow, CControlWinTraits>::Create`.

Reply:
This is a template class. Overriding is difficult.